frontend/src/views/Instructors/InstructorsList/InstructorsList.tsx

```diff
@@ -74,7 +86,16 @@ export const InstructorsList = () => {
                 </TableCell>
                 <TableCell>{row.user.email}</TableCell>
                 <TableCell>{row.user.phoneNumber}</TableCell>
-                <TableCell />
+                <TableCell>
+                  {row.instructorsQualifications.reduce(
```

Comment:
Personally I wouldn't populate the instructor's `instructorsQualifications` on the backend. I'd return an array of ids and then map them on frontend (by making another call to `/api/drivers-license-categories` and using the response to display the name).

IMO this approach will make the BE code a bit simpler (no need for creating separate DTOs for version with ids and actual qualification values) but could potentially make it work a tiny bit faster (no need for `JOIN`s for each instructor on the BE, but one `SELECT` on the whole table).

<details>
  <summary>
<i>Additional points for making the mapping be <code>O(1)</code> instead of <code>O(n)</code> by utilising the map that maps <code>id → name</code> instead of iterating over all qualifications each time.</i>
 </summary>
 It will be a minuscule performance gain considering the amount of that that we have, but that should also make the code a bit readable <br /> <br />
Here's <a href="https://www.typescriptlang.org/play?target=99#code/MYewdgzgLgBAJgQygmBeGBvAUDGAiYJAUwHMQAnASyIjwC4YBtHXTF1-SuemAJgBp2rPGAQBbIjzwBBALJ4hAX0EdsHXHi48AzCvUjxkhjICMCjsqFr9WhgBY9HAxKnTe51pdVCNtmAFZHYVEXY2kPXC9Wayc-ADYgjRCjfAAhMyUgmOE-AHZE-GSpVIiYKNxs324GAA4C5xS8VIBqAFFS8rZ1TmqYAE56ouMAYQyLLJ8enhMABkHDKVG2jonuzV6TE3nQ-GGVq0n16YFDod3lzIO1vxNdU4XjABExz1WbDYd7nbxni-Gr97TQJfRqPfbea4bBIgqSPP6vAGxDb5GHGAAq4OihxudVR+DR5HEAHcEAArDosAC6WEUWCwoEgsEQyAAknBZAgAA4MABKRFA5DgAB4wABXMQAIyI5H4MGgVDAJAAfGgYAB5CWk-lQAB0ADNyCAxK0wFAqDQABTMhA6whQUgUagQHViLkWi0YGBcWXJMoASjQKsY3pgyUpfr9dIZEBAABsiDrYyASFakAg2RzOX6ANxAA">my solution</a> for that if you don't figure it out yourself (or if you don't want to spend too much time on it 😉)
</details>